tequilapi/client/client.go

```diff
@@ -734,3 +736,24 @@ func (client *Client) UpdateTerms(obj contract.TermsRequest) error {
 	defer resp.Body.Close()
 	return nil
 }
+
+// FetchConfig - fetches current config
+func (client *Client) FetchConfig() (map[string]interface{}, error) {
+	resp, err := client.http.Get("config", nil)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	var unmarshaled map[string]interface{}
+	err = json.Unmarshal(bodyBytes, &unmarshaled)
+
+	config := unmarshaled["data"].(map[string]interface{})
```

Comment:
I think there are two ways this can panic:
1. If we get an error on json.Unmarshal this will panic as there might be no such thing as `data` resulting in a nil (check for error before trying to cast)
2. If for some reason structure changes and `data` is no longer a `map[string]interface{}` (`config, ok := ..` )